pkg/assume/entrypoint.go

```diff
@@ -44,6 +44,7 @@ func GlobalFlags() []cli.Flag {
 		&cli.StringFlag{Name: "sso-region", Usage: "Use this in conjunction with --sso, the sso-region"},
 		&cli.StringFlag{Name: "account-id", Usage: "Use this in conjunction with --sso, the account-id"},
 		&cli.StringFlag{Name: "role-name", Usage: "Use this in conjunction with --sso, the role-name"},
+		&cli.StringFlag{Name: "mfa-token", Usage: "Use this to bypass MFA prompt"},
```

Comment:
the word bypass makes it sound like assume is some how getting around the mfa requirement

```suggestion
		&cli.StringFlag{Name: "mfa-token", Usage: "Provide your current mfa token for the role you are assuming to skip being prompted"},
```